convert ALL maps to TGM format, remove dirty variables

nervere opened the pull request:
## What this does
All maps not previously in TGM format have been converted. All maps with dirty variables have had them removed.
Exceptions:
- Both maps in the "broken" folder, since they're broken and I can't open them without StrongDMM deleting a lot of typepaths that have since been removed. (jungle.dmm and zresearchlabs.dmm)
- Three maps that @Optimism333 is currently working on, as he has converted them in #34459 (habitation.dmm, research.dmm, mothership_lab.dmm)

## Why it's good
Standardizes map formatting, removes useless data.

[system]

Eneocho commented:
You know what they say about consequences. Something about them being unforeseen or something.

nervere commented:
> You know what they say about consequences. Something about them being unforeseen.

It'll be fine.

Exxion commented:
What the fuck? I was looking through the changes for some reason and saw some removals of `dir = 4`, which *shouldn't* be dirty
But apparently, `/obj/machinery/door` overrides the default value of `dir`? Why?

Exxion commented:
Did you intentionally make any changes to any of these maps? A few of them have some minor changes, besides updating typepaths and removing redundant variables. I've found two maps with some changes, but I haven't looked through all of them.

nervere commented:
> What the fuck? I was looking through the changes for some reason and saw some removals of `dir = 4`, which _shouldn't_ be dirty But apparently, `/obj/machinery/door` overrides the default value of `dir`? Why?

Should I do something about this? I wouldn't mind going through all the maps again if a change is necessary.



> Did you intentionally make any changes to any of these maps? A few of them have some minor changes, besides updating typepaths and removing redundant variables. I've found two maps with some changes, but I haven't looked through all of them.
> 
> EDIT: Some of the changes I was seeing were spurious, but also some weren't. Specifically, a firefighter Ripley has mysteriously appeared where one previously was not on each of `spacebattle.dmm` and `blackmarketpackers.dmm`, and also on the latter, a piece of huge mushroom meat has moved itself from one place to another.

I made three changes to maps, all of which were due to missing typepaths:
- I replaced the broken firefighter ripley typepath on spacebattle.dmm with the new typepath for it.
- I added three huge mushroom slices to blackmarketpackers.dmm because the old typepath for it was removed.
- I added a death commando mask to wine_cellar.dmm, same story, old typepath changed

For spacebattle.dmm, I didn't have a good way of figuring out where the Ripley was supposed to be, since StrongDMM just deletes whatever has an invalid typepath, so I put it next to the Odysseus in the spawn area since it seemed to make the most sense.

The extra Ripley on blackmarketpackets.dmm was NOT intentional, seems to have been an error. Seems I just placed it somewhere random by accident. Anyway, latest commit removes that.

nervere commented:
I'm just now realizing I could've just edited the .dmm file directly to fix the typepaths but that didn't occur to me at the time.

Exxion commented:
> For spacebattle.dmm, I didn't have a good way of figuring out where the Ripley was supposed to be, since StrongDMM just deletes whatever has an invalid typepath, so I put it next to the Odysseus in the spawn area since it seemed to make the most sense.

Yes, and there's supposed to be one there, but a second one appeared somewhere else.

There are supposed to be three pieces of mushroom meat in `blackparketpackers.dmm`, but one of them has moved from a normal floor to a carpet.

nervere commented:
I see. I'm just gonna reset my changes to these files and then edit the broken typepaths manually, which is what I should've done from the beginning.

nervere commented:
Right, it's been done properly now. Thank you for bringing that to my attention.

PrimeDSS13 commented:
we're not TG(M)
